src/cmds/scripts/Makefile.am

```diff
@@ -43,8 +43,6 @@ dist_libinit_DATA = \
 	limits.pbs_mom.compat \
 	limits.post_services \
 	limits.post_services.compat \
-	sgigenvnodelist.awk \
-	sgiICEvnode.sh \
 	sgiICEplacement.sh
```

Comment:
Do we still need sgiICEplacement.sh?

Reply:
No.

Reply:
After a discussion, it was decided to keep the sgiICEplacement.sh file.
And also to not remove the sgiICEvnode.sh file.

Reply:
I'm reversing my opinion on this. We should keep sgiICEplacement.sh. Also, we should put back sgiICEvnode.sh, so that a site is running on HPE ICE 8600, they have the option that if they don't run the pbs_cgroup hook, then they can generate manually the vnode definitions file and placement sets using the sgi* scripts...